config/gtasa/constants.xml

```diff
@@ -426,9 +460,7 @@
       <Constant Name="PEDTYPE_MISSION6"/>
       <Constant Name="PEDTYPE_MISSION7"/>
       <Constant Name="PEDTYPE_MISSION8"/>
-      <Constant Name="PEDTYPE_LAST_PEDTYPE"/>
```

Comment:
ditto?

Reply:
Yeah, the last value for this enum is PEDTYPE_MISSION8
![image](https://github.com/user-attachments/assets/702f605c-6b34-4a79-9335-bf3bcd40b16e)